torch/_torch_docs.py

```diff
@@ -7754,3 +7754,82 @@ def merge_dicts(*dicts):
     >>> g_cpu.device
     device(type='cpu')
 """)
+
+add_docstr(torch.searchsorted,
+           r"""
+searchsorted(sorted_sequence, values, out_int32=False, right=False, out=None) -> Tensor
+
+Returns a new tensor, same size as :attr:`values`, containing the result of applying
```

Comment:
Ditto here, the numpy version of this function https://docs.scipy.org/doc/numpy/reference/generated/numpy.searchsorted.html 's documentation reads better to me

Reply:
same as above, will take a look at

Reply:
updated